src/optimization/DAG.h

```diff
@@ -0,0 +1,34 @@
+//
+// Created by nomaddo on 17/12/18.
+//
```

Comment:
Can you change the license-info (in `DAG.cpp`) too to fit the style of the other files, which then would be:

    /*
     * Author: nomaddo
     *
     * See the file "LICENSE" for the full license governing this code.
     */